### PR TITLE
feat: Adds Test Groups to Users

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2020-03-04 at 11 25 02 AM

### What does this PR do, or why is it needed?

Allows us to control app tests features per on a per user basis from Rock

### How do I test this PR?

Add yourself to the groups and make sure they show up with this query:

```
{
  currentUser {
    profile {
      featureGroups {
        name
      }
    }
  }
}
```

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings
- [x] Upload Screenshots/GIF(s) if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

---

> The purpose of a PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
actually I'm gonna rethink this strategy

File: src/data/people/schema.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ const peopleSchema = gql`
 
   extend type Person {
     impersonationParameter: String! @deprecated(reason: "No longer used.")
-    isGroupLeader: Boolean
```
Comment (richarddubay) [inline src/data/people/schema.js]: If we remove `isGroupLeader` won't the group leader link on the Connect screen no longer work?

Comment (redreceipt) [reply src/data/people/schema.js]: I moved it to the groups schema

Comment (richarddubay) [reply src/data/people/schema.js]: Haha, of course. Plus I just could test in the client and make sure it works. 🤦‍♂️

**Pull request merged**